roles/pulp-api/tasks/signing_service.yml

```diff
@@ -11,6 +11,12 @@
         status: "False"
         lastTransitionTime: "{{ lookup('pipe', 'date --iso-8601=seconds') }}"
 
+- name: Set GPG initContainer image
+  set_fact:
+    gpg_init_container_image: >-
+      {{ lookup('env', 'GPG_INIT_CONTAINER_IMAGE') |
```

Comment:
Can we re-name this to `RELATED_IMAGE_GPG_INIT_CONTAINER` ?  The requirement is that the env var be prefixed with `RELATED_IMAGE_`.